### PR TITLE
ERIC API GET Method Implementation

### PR DESCRIPTION
**Pull request by Bera0422**
This PR includes the implementation of the GET method that utilizes the ERIC API.

### CODE REVIEW

**Comment by hakanaktas0** [comment]
You should omit binary files and sqlite file from your commits.

File: practice_app/api/urls.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
 from django.urls import path
 
-urlpatterns = [
+from . import views
 
-]
\ No newline at end of file
+app_name = "api"
+urlpatterns = [
+    path('eric/', views.eric_index, name='eric_index'),
```
Comment (hakanaktas0) [inline practice_app/api/urls.py:7]: I think you should omit the index page and just provide the GET function in this url.

**Review by hakanaktas0 — CHANGES_REQUESTED** [review]
Binary files and sql file should be removed.

File: practice_app/api/urls.py (hunk 1 above)
Comment (Bera0422) [reply practice_app/api/urls.py]: I omitted the index page. The GET function can be utilized from this 'eric/' endpoint.

**Comment by Bera0422** [comment]
> Binary files and sql file should be removed.

I removed them.

**Comment by hakanaktas0** [comment]
Can you also exclude the files with unnecessary changes? Some files use ' instead of ". One binary file also remains.

**Comment by Bera0422** [comment]
Done.

File: practice_app/api/views.py [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+    response_fields = "&fields=id AND title AND author AND publicationdateyear AND url AND description AND source"
+    endpoint = baseURL + "?search=title:" + search_title + "&rows=" + rows + response_fields
+
+    response = requests.get(endpoint)
+
+    if response.status_code == 200:
+        papers = response.json()['response']['docs']
+        
+        i = 0
+        for paper in papers:
```
Comment (defabdullah) [inline practice_app/api/views.py:35]: this logic seems like unnnecessary in this code

Comment (Bera0422) [reply practice_app/api/views.py:35]: I think it is necessary for the position key.

**Review by hakanaktas0 — APPROVED** [review]
After resolving the conflicts this PR should be good to merge.

File: practice_app/api/views.py (hunk 2 above)
Comment (defabdullah) [reply practice_app/api/views.py:35]: Sorry. I thought that it doesn't affect papers variable but it does.

**Review by defabdullah — APPROVED** [review]

**Pull request merged**